### PR TITLE
Add docker run instructions to README

### PR DESCRIPTION
**Pull request by PrplHaz4**
Add docker run instructions to README, use host networking to avoid explicitly mapping a bunch of ports, and trace logging to document for any users that need to troubleshoot.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -245,6 +245,9 @@ branch gets it's own tag.
 
 ```bash
 docker pull quantumentangledandy/neolink
+
+# remove `-e "RUST_LOG=trace"` to run in production mode
+docker run --name neolink --network host --restart=unless-stopped -d -e "RUST_LOG=trace" --volume=$PWD/config.toml:/etc/neolink.toml quantumentangledandy/neolink
```
Comment (QuantumEntangledAndy) [inline README.md:250]: Trace level log is a bit much I think, so I would prefer this to read `debug`

**Comment by QuantumEntangledAndy** [comment]
What was your motivation behind this? I'm not sure about the host mode by default. I don't run it like that myself, instead I supply full address information in my config so it can connect without the broadcasts.

**Comment by PrplHaz4** [comment]
> What was your motivation behind this? I'm not sure about the host mode by default. I don't run it like that myself, instead I supply full address information in my config so it can connect without the broadcasts.

This note was on the original repo.  I started using host networking mainly to rule that out as a source of problems when testing a battery cam (Reolink Go PT).
> If you use a battery-powered camera (or other UDP-only camera) you will need to either use --net=host or setup a [macvlan](https://docs.docker.com/network/macvlan/) for the docker image that supports UDP broadcast. This is because UDP requires that udp broadcast messages are transmitted across the docker network interface, however this is https://github.com/docker/for-linux/issues/637

**Comment by QuantumEntangledAndy** [comment]
I see. Well I might want to add a note about how to connect. Host mode should be mostly optional now since we can connect to udp without using broadcasts now.

But I'll merge this for now and add it to my todo list :)

**Comment by QuantumEntangledAndy** [comment]
Thanks for the contribution

**Pull request merged**